docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -30,60 +30,63 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 .. tab:: Software Requirements
 
   * `CMake 3.13 or higher <https://cmake.org/download/>`_ (choose "macOS 10.13 or later"). Add `/Applications/CMake.app/Contents/bin` to path (for default install). 
-  * `Python 3.6 - 3.9 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6 - 3.10). Install and add to path.
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.10 are recommended.
+  * `Python 3.6 - 3.10 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6 - 3.10). Install and add to path.
+     * Note that OpenVINO is gradually phasing out the support for Python 3.6. Python 3.7 - 3.10 are recommended.
```

Comment:
drop 3.10 here. We are going to support it since 22.3

Reply:
Yeah, this is for master, so it's for 2022.3. Actually this change is done in another PR: https://github.com/openvinotoolkit/openvino/pull/13243